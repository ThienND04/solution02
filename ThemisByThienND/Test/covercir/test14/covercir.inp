4
48864 43302  33251
58791 57625  42803
57266 2925  10532
41556 115  29029
