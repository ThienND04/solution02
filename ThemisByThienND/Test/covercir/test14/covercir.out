29225
34253
32435
23489
